data/test/scenarios/wml_tests/ScenarioWML/EventWML/events-test_filter_ability.cfg

```diff
@@ -296,3 +296,143 @@
 )}
 
 #undef FILTER_ABILITY_TEST
+
+##
+# Common code for the tests in this file.
+##
+# Actions:
+# Give both Alice and Bob 100% chance to hit.
+# Give Alice a drains ability which is only active during liminal times of day and to value 25(defined in hardcoding by default).
+# During Alice's turn, move Alice next to Bob, and have Alice attack Bob.
+# During Bob's turn, have Bob attack Alice.
+##
+#define FILTER_ABILITY_TEST_WITHOUT_VALUE
+    [event]
+        name=start
+        # Make sure the attacks hit
+        {FORCE_CHANCE_TO_HIT (id=bob) (id=alice) 100 ()}
+        {FORCE_CHANCE_TO_HIT (id=alice) (id=bob) 100 ()}
+        [modify_unit]
+            [filter]
+            [/filter]
+            # Make sure they don't die during the attacks
+            [status]
+                invulnerable=yes
+            [/status]
+        [/modify_unit]
+        [object]
+            silent=yes
+            [effect]
+                apply_to=new_ability
+                [abilities]
+                    [drains]
+                        [filter]
+                            [filter_location]
+                                time_of_day=neutral
+                            [/filter_location]
+                        [/filter]
```

Comment:
Is there a reason this filter needs to be part of these tests?

Reply:
no i removed this